src/proto_nd_flow/reco/charge/raw_event_generator.py

```diff
@@ -154,17 +154,25 @@ def _convert_mc_truth_tracks(self, tracks):
     def _convert_mc_truth_trajectories(self, traj):
         traj_copy = traj.copy()
         for field in ('xyz_start', 'xyz_end'):
-            traj_copy[field][:, 0] = traj[field][:, 2] * units.mm
-            traj_copy[field][:, 1] = traj[field][:, 1] * units.mm + 218.236
-            traj_copy[field][:, 2] = traj[field][:, 0] * units.mm
+            traj_copy[field][:, 0] = traj[field][:, 2] * units.cm
+            traj_copy[field][:, 1] = traj[field][:, 1] * units.cm + 21.8236
+            traj_copy[field][:, 2] = traj[field][:, 0] * units.cm
         for field in ('pxyz_start', 'pxyz_end'):
-            traj_copy[field][:, 0] = traj[field][:, 2] * units.MeV
-            traj_copy[field][:, 1] = traj[field][:, 1] * units.MeV
-            traj_copy[field][:, 2] = traj[field][:, 0] * units.MeV
+            traj_copy[field][:, 0] = traj[field][:, 2] * units.keV
```

Comment:
What is the rationale for keV instead of MeV here?

Reply:
Running ndflow with the units set to MeV for the trajectories gives output that is 1000x too large for the momentum (e.g. pxyz_start). The attached plot shows the muon initial momentum distribution and note the axis scale, the peak is at ~5E6 MeV or 5 TeV.

Changing the units to keV fixes this, however looking at the larndim files the trajectory momentum seems to be correctly in MeV... so I'm not sure if there is some other conversion being applied to the units.

![image](https://user-images.githubusercontent.com/33762574/233474663-a560a736-2acb-4f1d-8742-ae82f12af48e.png)

Reply:
I will also note here (so as not to forget) that I need to double check the y-offset and the position units.